evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -188,6 +188,23 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate "Copy" %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+            <confirmation-modal type="submit" id="exam_creation_modal" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
```

Comment:
The `id` of these modals will be duplicated when there is more than one applicable evaluation - I think we don't need the id anyways, in that case let's just remove it

Reply:
I do not think we can remove it. Without id it throws: `ValueError: "<Evaluation: Database Systems I – Exam>" needs to have a value for field "id" before this many-to-many relationship can be used.`

Reply:
Sorry, that wasnt very clear - I was referring to the id of the html element, that is, the `id=exam_evaluation_modal` part